libsrc/geos-common/disk/calcblksfree.s

```diff
@@ -13,6 +13,10 @@
             .include "geossym.inc"
         
 _CalcBlksFree:
+        lda #>curDirHead
+        sta r5H
+        lda #<curDirHead
+        sta r5L
```

Comment:
While the code is correct it doesn't fit the usual cc65 style. In cc65 Asm code we use the combination of A and X to represent 16-bit values (especially pointers). You'll often find the term AX to represent that (A for the low byte and X for the high byte). And when moving those 16-bite values around we start with the low byte. So the code here should read

```
lda #<curDirHead
ldx #>curDirHead
sta r5L
stx r5H
```

If you provide an update to this pull request I can merge it as-is.